ui/components/app/confirm/header/header.scss

```diff
@@ -0,0 +1,11 @@
+.confirm_header {
+  &__wrapper {
+    box-shadow: 0 2px 16px 0 #0000001a;
+    height: 64px;
+  }
+
+  &__avatar-network {
+    margin-left: -12px;
+    margin-top: -4px;
+  }
+}
```

Comment:
```suggestion
}

```

<img width="1471" alt="Screenshot 2023-12-15 at 4 16 22 PM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/dff7ab27-d595-4b56-b841-de4b5d77bf4b">

there's a lint error here. maybe this suggestion would fix it